Flagsmith-engine/Engine.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.Linq;
+using System.Collections.Generic;
+using Flagsmith_engine.Exceptions;
+using System;
+using Flagsmith_engine.Interfaces;
+using Flagsmith_engine.Segment;
+using Flagsmith_engine.Environment.Models;
+using Flagsmith_engine.Feature.Models;
+using Flagsmith_engine.Identity.Models;
+using Flagsmith_engine.Trait.Models;
+
+namespace Flagsmith_engine
```

Comment:
Perhaps this is a .NET convention but it feels a bit odd that the namespace is `Flagsmith_engine` instead of `FlagsmithEngine` ?

Reply:
Yes, will correct that to FlagsmithEngine .